server/static/script.js

```diff
@@ -138,10 +140,10 @@ async function pourBeer() {
     const grid = document.getElementById('mainContent');
     grid.innerHTML = SUCCESS_CONTENT;
 
-    // Reset back to the home screen after one second.
+    // Reset back to the home screen after pouring
     setTimeout(() => {
         showHome();
-    }, 3000);
+    }, 18000); // = POUR_TIME
```

Comment:
I think we can just couple these together - to ensure they don't get de-synced.

```suggestion
    }, POUR_TIME);
```

Reply:
POUR_TIME is in python though?

Reply:
Oh, d'oh, indeed it is.

I had the idea to pass it down in the response to `/pour`. The web UI could also show a countdown in seconds for the remaining pour time :)

Reply:
yes, sounds nice.... uhh feel free to PR? :P